greek/take_the_fork/put_a_spell/put-a-spell.md

```diff
@@ -0,0 +1 @@
+Αρχίζεις να απαγγέλλεις δυνατά το μαγικό ξόρκι που σου έμαθα ο παππούς σου "Legilimens.. Legilimens.. Legilimens!!!". Πλέον έχεις διεισδύσει στο μυαλό των ζαχαρωτών και μπορείς να ελέγξεις τις σκέψεις τους. Ωρα να δράσεις... 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.